live_view_studio/lib/live_view_studio_web/router.ex

```diff
@@ -0,0 +1,59 @@
+defmodule LiveViewStudioWeb.Router do
+  use LiveViewStudioWeb, :router
+
+  pipeline :browser do
+    plug :accepts, ["html"]
+    plug :fetch_session
+    plug :fetch_live_flash
+    plug :put_root_layout, {LiveViewStudioWeb.LayoutView, :root}
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+  end
+
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  scope "/", LiveViewStudioWeb do
+    pipe_through :browser
+
+    get "/", PageController, :index
+
+    #define live route using the live macro -> takes a path and a LiveView module to use
+    live "/light", LightLive
```

Comment:
@keliumJU what is the difference between `live` and a regular route as get or post?

Reply:
I think the main difference is that live is a webSockets request instead get, post, update, delete ... are HTTP requests